extensions/amp-gfycat/0.1/amp-gfycat.js

```diff
@@ -87,14 +87,14 @@ class AmpGfycat extends AMP.BaseElement {
 
   /** @override */
   createPlaceholderCallback() {
-    const placeholder = this.win.document.createElement('amp-img');
+    const placeholder = this.win.document.createElement('img');
```

Comment:
Do you think we should move this over to the html helper?

Reply:
Not necessarily. The static portion is very small.